plugins/outputs/datadog/README.md

```diff
@@ -46,11 +54,9 @@ field key with a `.` character.
 Field values are converted to floating point numbers.  Strings and floats that
 cannot be sent over JSON, namely NaN and Inf, are ignored.
 
-We do not send `Rate` types. Counts are sent as `count`, with an
-interval hard-coded to 1. Note that this behavior does *not* play
-super-well if running simultaneously with current Datadog agents; they
-will attempt to change to `Rate` with `interval=10`. We prefer this
-method, however, as it reflects the raw data more accurately.
+Enabling the `should_rate_counts` will convert `count` metrics to `rate`
+and divide it by the `rate_interval`. This will allow telegraf to run
+alongside current Datadog agents.
```

Comment:
I think you should be even more explicit and that that it requires the counter `metric_type` tag from the statsd input (or count from a histogram/timing). A user couldn't use any other plugin without adding that tag right?

Reply:
That's correct, my development has been based only on `inputs.statsd` since I believe that the be the most common approach. i.e. instrumenting application with Dogstatsd library, and submitting to a StatsD receiver in either the Datadog agent or elsewhere (Telegraf in this case).

I'll add a commit to make this clearer.

Reply:
Added in https://github.com/influxdata/telegraf/pull/15702/commits/ab1fa8738b4688c50fa4e06eda848809ff33f357